File: NPB-CPP-cadna/cadna_c-3.1.11/examplesFortran/ex4_cad.f90
      program ex4_cad
      use cadna
      implicit type (double_st) (a-h,o-z)
      call cadna_init(-1)
      !
      print *,'A second order recurrent sequence with CADNA:'
      a = 5.5d0
      b = 61.d0/11.d0
      do i=3,30
        c = b
        b = 111.d0 - 1130.d0/b + 3000.d0/(a*b)
        a = c
        write(*,'(a2,i2,a3,a23)')'U(',i,') = ',str(b)
      enddo
      write(*,*)'The exact limit is 6.'
      call cadna_end()
!
      end

File: NPB-CPP-cadna/cadna_c-3.1.11/examplesC/ex4_cad.cc
#include <cadna.h>
#include <math.h>
#include <stdio.h>

using  namespace std;

int main()
{
  int i;
  double_st a,b,c;

  cadna_init(-1);

  printf("-------------------------------------\n");
  printf("| A second order recurrent sequence |\n");
  printf("| with CADNA                        |\n");
  printf("-------------------------------------\n");

  a = 5.5;
  b = 61./11.; 
  for(i=3;i<=30;i++){
    c = b;
    b = 111. - 1130./b + 3000./(a*b);
    a = c;
    printf("U(%d) = %s\n",i,strp(b));
  }
  printf("The exact limit is 6.\n");

  cadna_end();
}






